src/test/java/com/puppycrawl/tools/checkstyle/XMLLoggerTest.java

```diff
@@ -382,6 +382,14 @@ public void testFileOpenTag()
 
     }
 
+    @Test
+    public void testVerifyLogger()
+            throws Exception {
+        final String className = "InputXMLLogger.java";
+        final String xmlReport = "ExpectedXMLLoggerWithChecker.xml";
+        verifyWithInlineConfigParserAndXmlLogger(className, xmlReport);
```

Comment:
please make one more  method like this that takes array of target files.

`verifyWithInlineConfigParserAndXmlLogger(inputFileWithConfig, xmlReport, targetFiles);`

where targetFiles is array files for Checker to run on, such attay should have two files `InputXMLLogger.java` (duplicate presence).
By Checkstyle API, we allow to send to Checker same file twice.
Expected Behavior is this case not clear, and we can accept dulicate tag at you have at src/test/resources/com/puppycrawl/tools/checkstyle/xmllogger/ExpectedXMLLoggerDuplicatedFile.xml, so ExpectedXMLLoggerDuplicatedFile.xml just should be upated to have duplcate validation from InputXMLLogger.java